client/services/code_evaluators/PythonCodeRunnerService.js

```diff
@@ -77,7 +77,7 @@ tie.factory('PythonCodeRunnerService', [
 
           // The run was successful.
           return CodeEvalResultObjectFactory.create(
-            code, outputLines.join('\n'), correctnessTestResults,
+            code, outputLines, correctnessTestResults,
```

Comment:
If we are doing this then perhaps we might need to change the output property in CodeEvalResult to outputLines.